lib/manager/cargo/artifacts.js

```diff
@@ -36,26 +36,33 @@ async function getArtifacts(
     logger.debug('Updating ' + lockFileName);
     for (let i = 0; i < updatedDeps.length; i += 1) {
       const dep = updatedDeps[i];
-      // Unambiguous specification of a crate would be <crate_name>:<locked_version>,
-      // might be required, because it is possible to have multiple versions of the
-      // same dependency in the same project.
-      // see cargo help pkgid for additional details
-      // NOTE: Complex version requirements like '=0.2.0' might be a problem
-      const spec = dep;
+      // const spec = dep;
       const cmd =
-        'cargo update --manifest-path ' + localPackageFileName + ' --package ';
+        'cargo update --manifest-path ' + localPackageFileName + ' --package';
       // if (config.binarySource === 'docker') {
       //   cmd = 'docker run renovate/cargo update -p';
       // }
       const startTime = process.hrtime();
-      ({ stdout, stderr } = await exec(`${cmd} ${spec}`, {
-        shell: true,
-      }));
+      try {
+        ({ stdout, stderr } = await exec(`${cmd} ${dep}`, {
+          shell: true,
+        }));
+      } catch (err) {
+        logger.debug(
```

Comment:
Are you sure you can do this for *all* errors. I assume you should probably against certain error messages only.